package.json

```diff
@@ -32,5 +32,8 @@
   "scripts": {
     "boosh": "smoosh make ./build.json",
     "test": "node ./test.js"
+  },
+  "dependencies": {
+    "xhr2": "0.1.0"
```

Comment:
this is going to add the library to the client. if you build this with `ender` or `browserify` i'm not so sure this library will work anymore in the browser (which is its original intention)